tests/unit/models/test_utils.py

```diff
@@ -11,3 +32,24 @@ class TestUtils(TestCase):
     def test_kwargs_req(self):
         with self.assertRaises(XRPLModelException):
             IssuedCurrency(currency, issuer)
+
+    def test_throws_if_positional_args_mixed_with_non_positional_args(self):
+        with self.assertRaises(XRPLModelException):
+            ChannelAuthorize(
```

Comment:
Please use simpler/more common API calls - ideally one request and one transaction.